nginx.conf

```diff
@@ -0,0 +1,36 @@
+# Run in the foreground
+daemon off;
+
+# Keep a single process, we can scale the number of container in k8s instead of tuning workers here
+worker_processes  1;
+
+# Send anything from info to errors to stdout
+error_log /dev/stdout info;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    access_log /dev/stdout;
+
+    server {
+       listen 8080;
+       root /home/nginx/recovery;
+       location = /health {
```

Comment:
Why a separate health endpoint? If you get a 200 from the recovery endpoint isn't it the same thing?

Reply:
yeah that's a good point. I'll remove it.

Reply:
You should *not* do this, the body of health checks is reported in kubernetes logs.
Try and have an endpoint that returns nothing other than e.g. `OK`

Reply:
Ah. Will add this back then!